### ISSUE TITLE
Show 3D-View Extent on 2D Map

### ISSUE THREAD

**Issue by Saijin-Naib** [qgis/QGIS#32975]
**Feature description.**
I would love to see an option to show the 3D View extent/center on our 2D Map canvas, as this could really help contextualize what the cone of vision is for the 3D View, as well as assist the user in aligning their cone of view with the aid of the 2D Map's clear view of the scene at nadir.

**Additional context**
![image](https://user-images.githubusercontent.com/19295950/69250345-8184c100-0b7d-11ea-89dd-63e7eeae5339.png)

This quick/simple mock-up shows something along the lines of what I'm asking for. We have the show view center dot from the 3D View, as well as a cone of vision. Ideally (and likely way complex/out of scope), the cone of vision would be occluded by the terrain/buildings, and you'd see where it actually goes to and what is above the cone)

**Comment by roya0045** [comment]
This is a good idea but I think it would be better to have a 3D 'atlas' where you could have the viewpoint in the overview 2d map.

**Comment by Saijin-Naib** [comment]
> This is a good idea but I think it would be better to have a 3D 'atlas' where you could have the viewpoint in the overview 2d map.
I'm not quite sure I understand what you're proposing. I've also proposed an analogous feature for the 3D View in Issue #32976, which would put the 2D Map view extent into the 3D View.

My idea is to help the user visualize where they are in space when viewing both maps side-by-side. I really grew to love this type of view cone from my time using GeoSetter to georefrence sUAS imagery I had collected, and I think this type of addition to QGIS's 3D view would be invaluable.
